salt/metalk8s/orchestrate/deploy_node.sls

```diff
@@ -52,7 +52,14 @@ Cordon the node:
     - kubeconfig: {{ kubeconfig }}
     - context: {{ context }}
 
-# TODO: Call drain
+Drain the node:
+  metalk8s_drain.node_drained:
+    - name: {{ node_name }}
+    - ignore_daemonset: True
+    - delete_local_data: True
+    - force: True
+    - kubeconfig: {{ kubeconfig }}
+    - context: {{ context }}
```

Comment:
This one should require `Cordon the node` and `Run the highstate` should require the Drain